`--features alloc` no longer compiles on nightly-2018-07-07

```console
$ cargo build --no-default-features --features alloc

error[E0432]: unresolved import `alloc::BTreeMap`
   --> serde/src/lib.rs:196:21
    |
196 |     pub use alloc::{BTreeMap, BTreeSet, BinaryHeap, LinkedList, VecDeque};
    |                     ^^^^^^^^ no `BTreeMap` in the root

error[E0432]: unresolved import `alloc::BTreeSet`
   --> serde/src/lib.rs:196:31
    |
196 |     pub use alloc::{BTreeMap, BTreeSet, BinaryHeap, LinkedList, VecDeque};
    |                               ^^^^^^^^ no `BTreeSet` in the root

error[E0432]: unresolved import `alloc::BinaryHeap`
   --> serde/src/lib.rs:196:41
    |
196 |     pub use alloc::{BTreeMap, BTreeSet, BinaryHeap, LinkedList, VecDeque};
    |                                         ^^^^^^^^^^ no `BinaryHeap` in the root

error[E0432]: unresolved import `alloc::LinkedList`
   --> serde/src/lib.rs:196:53
    |
196 |     pub use alloc::{BTreeMap, BTreeSet, BinaryHeap, LinkedList, VecDeque};
    |                                                     ^^^^^^^^^^ no `LinkedList` in the root

error[E0432]: unresolved import `alloc::VecDeque`
   --> serde/src/lib.rs:196:65
    |
196 |     pub use alloc::{BTreeMap, BTreeSet, BinaryHeap, LinkedList, VecDeque};
    |                                                                 ^^^^^^^^ no `VecDeque` in the root
```

--- a/serde/src/lib.rs
+++ b/serde/src/lib.rs
@@ -193,7 +193,7 @@ mod lib {
     pub use std::sync::{Arc, Weak as ArcWeak};
 
     #[cfg(all(feature = "alloc", not(feature = "std")))]
-    pub use alloc::{BTreeMap, BTreeSet, BinaryHeap, LinkedList, VecDeque};
+    pub use alloc::collections::{BTreeMap, BTreeSet, BinaryHeap, LinkedList, VecDeque};
     #[cfg(feature = "std")]
     pub use std::collections::{BTreeMap, BTreeSet, BinaryHeap, LinkedList, VecDeque};
 

